oak/server/loader/BUILD

```diff
@@ -66,7 +66,6 @@ container_image(
     files = [
         "oak_runner",
         "//examples/certs/docker:ca.pem",
-        "//examples/certs/docker:docker.key",
```

Comment:
Is it possible to remove the `.pem` files too? It does not seem quite right to include example certificates in the runner image.

Reply:
For consistency I will also provision them as secrets, even though they are not considered secret.